data/MuhammadRabi.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "Muhammad Rabi",
+  "type": "personal",
+  "bio": "I'm an Autodidact Front-end web developer, seeking contribution to open source projects",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/MuhammadRabi",
+      "icon": "FaGitHub"
```

You're probably using an invalid icon here. It should be `FaGithub`

```suggestion
      "icon": "FaGithub"
```